modules/oadp-ceph-cephfs-back-up-dba.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-ceph-cephfs-back-up-dba_{context}"]
+= Creating a DPA for use with CephFS storage
+
+You need to create a Data Protection Application (DPA) CR before you use the OpenShift API for Data Protection (OADP) 1.2 Data Mover to back up and restore data using CephFS storage.
```

Comment:
Thanks -- FYI, the link sent me to IBM documentation, which didn't have a link to the style guide.  Maybe you have a separate way to log on. I looked the issue up on my downloaded version of the guide and see your point.

Reply:
@michaelryanpeter Since I wanted to start the intro with "you," I switched the verb to "must." I think I wanted a softer touch at first, but decided "must" was not particularly harsh in this context.